### PR TITLE
✏️  Fix typos and clarifies that port must be an int

### PR DESCRIPTION
**Pull request by parry84**
Fix a few typos into README.md.

Moreover I added an explicit port, since for me it took a while to discover that it must be not be a string.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,7 +162,7 @@ defmodule Myapp.Producer do
   alias Amqpx.Gen.Producer
 
   def send_payload(payload) do
-    Producer.publish("myexchange", "my.routing_key", payload)
+    Producer.publish("my_exchange", "my_queue", payload)
```
Comment (michelangelomo) [inline README.md:165]:
```suggestion
    Producer.publish("my_exchange", "my_exchange_routing_key", payload)
```

File: README.md [hunk 2]
```diff
@@ -84,16 +85,17 @@ WARNING: headers exchange binding not supported by library helpers functions
...
     }
+  ]
 
-config :myapp, Your.Handler.Module, %{
+config :myapp, Myapp.Consumer, %{
     queue: "my_queue",
     exchanges: [
       %{name: "amq.topic", type: :topic, routing_keys: ["my.routing_key1","my.routing_key2"], opts: [durable: true]},
-      %{name: "my_exchange", type: :direct, routing_keys: ["my_queue"]},
+      %{name: "my_exchange", type: :direct, routing_keys: ["my_queue"], opts: [durable: true]},
```
Comment (michelangelomo) [inline README.md:98]: Our typo :smile: 
```suggestion
      %{name: "my_exchange", type: :direct, routing_keys: ["my.direct_routing_key"], opts: [durable: true]},
```

**Comment by larrywax** [comment]
Hi @parry84, we'll be happy to merge and release this pr after you make the requested changes.
Thank you!

File: README.md (hunk 2 above)
Comment (larrywax) [reply README.md:98]: This is ok, is a direct exchange, so routing key and queue name must match

**Pull request merged**